### ISSUE TITLE
Delta score interpretation

### ISSUE THREAD

**Issue by yanwang271** [Illumina/SpliceAI#125]
Hi,

I was wondering how to interpret the prediction scores when the majority of people carrying the ALT allele. For example, SpliceAI predict ```T|RYR1|0.00|0.00|0.91|0.08|-28|-46|-2|-31``` for ```19:38958362 C>T```. If most people carry T while C is the minor allele, can I interpret the scores as DS_donor_loss = 0.91 and DS_donor_gain = 0.08 for ```19:38958362 T>C``` ?

Thanks!
Yan

**Comment by kishorejaganathan** [comment]
Yes, that interpretation is correct. In this particular example, splicing probability with T - splicing probability with C is 0.91. So for C>T it is donor gain of 0.91 and for T>C, it is donor loss of 0.91.
